apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/model/DBSortOrder.java

```diff
@@ -0,0 +1,25 @@
+package com.apollocurrency.aplwallet.apl.exchange.model;
+
+public enum DBSortOrder {
+    ASC("ASC"),
+    DESC("DESC");
+
+    String value;
+
+    DBSortOrder(String value) {
+        this.value = value;
+    }
+
+    public static DBSortOrder ordinal(int ordinal) {
```

Comment:
Resteasy also support static fromString or valueOf methods which accept String parameter and return desired value